### PR TITLE
Use .replace('.', '_') here as well?

### PR DESCRIPTION
**Pull request by kcbark**
I believe we should replace . with _ here as well as we do in list_get_jid ioc_list.py?
This unborked my jails (named www_test_com) after upgrading from iocage 1.1 to master.

### CODE REVIEW

**Review by skarekrow — CHANGES_REQUESTED** [review]
Yes we should, let's keep the single quotes though :P

File: iocage_lib/ioc_json.py [hunk 1]
```diff
@@ -1536,7 +1536,7 @@ def get_full_config(self):
         if state:
             ruleset = su.check_output(
                 [
-                    'jls', '-j', f'ioc-{conf["host_hostuuid"]}',
+                    "jls", "-j", f'ioc-{conf["host_hostuuid"].replace(".", "_")}',
```
Comment (skarekrow) [inline iocage_lib/ioc_json.py]:
```suggestion
                    'jls', '-j', f'ioc-{conf["host_hostuuid"].replace(".", "_")}',
```

**Review by skarekrow — DISMISSED** [review]

**Review by skarekrow — DISMISSED** [review]

**Comment by skarekrow** [comment]
Thanks @kcbark!

**Comment by skarekrow** [comment]
@kcbark The travis test will need to be fixed though before merging :)

**Comment by kcbark** [comment]
Ah, didn’t see that. I’ll fix :)

**Review by skarekrow — APPROVED** [review]

**Pull request merged**